module/create_connection.py

```diff
@@ -9,3 +11,29 @@ def create_connection(db_file: str) -> sqlite3.Connection:
     except Error as e:
         print(str(e))
     return conn
+
+
+def connect_and_execute(context: CallbackContext, chat_id: int, query: str, params: tuple, operation: str) -> Union[int, list, None]:
```

Comment:
I don't remember if python3.9 supports this
```suggestion
def connect_and_execute(context: CallbackContext, chat_id: int, query: str, params: tuple, operation: str) -> Optional[int | list]:
```